d2l/torch.py

```diff
@@ -991,11 +991,11 @@ class AdditiveAttention(nn.Module):
     """Additive attention.
 
     Defined in :numref:`sec_attention-scoring-functions`"""
-    def __init__(self, key_size, query_size, num_hiddens, dropout, **kwargs):
+    def __init__(self, num_hiddens, dropout, **kwargs):
```

Comment:
Please rerun `d2lbook build lib` after further code changes.

Reply:
It is done in the latest submission.